src/parse/ris_live.go

```diff
@@ -79,44 +80,43 @@ type RisLiveMessage struct {
 }
 
 // connects to ris live, starts go routine receiverHandler, manages connection and subscription
-func ParseRisLiveData(msgChannel chan common.Message, config *config.Configuration) error {
+func ParseRisLiveData(msgChannel chan common.Message, config *config.Configuration) {
 
 	fmt.Println("starting...")
 
+	//for each subscription
+	for _, subscription := range config.Subscriptions {
```

Comment:
A quality of life improvement here would be to use the index from `range`, i.e. `for i, sub := range ...`, and pass the index as a parameter down into `handleSubscription()`

the reason: `handleSubscription` calls `receiveHandler` which handles the BGP messages. At the moment, `receiveHandler` spits out a lot of information, and it's not clear by eye which subscription the messages relate to.

There's an open question on how to tidy all that terminal output (longer topic on refactoring into a logging library; future work!), but prepending the index value onto the Printf that writes `Parsed BGP Message` to terminal would help disambiguate which subsciption caught the message